cluster/manifests/skipper/deployment.yaml

```diff
@@ -227,12 +227,21 @@ spec:
           {{ end }}
 {{ end }}
         resources:
+{{ if and (eq .Cluster.ConfigItems.enable_dedicate_nodepool_skipper "true") (eq .Cluster.ConfigItems.skipper_ingress_binpack "true") }}
+          limits:
+            cpu: "{{ sumQuantities .Cluster.ConfigItems.skipper_ingress_cpu (printf \"-%s\" .Cluster.ConfigItems.teapot_admission_controller_daemonset_reserved_cpu) (printf \"-%s\" .Cluster.ConfigItems.kubelet_system_reserved_cpu) (printf \"-%s\" .Cluster.ConfigItems.kubelet_kube_reserved_cpu) }}"
```

Comment:
minor: could be a template-local var like
```
{{ $internal_version := "v0.13.208-261" }}
```

Reply:
Not sure if it is any better
```
{{ split(printf "%s -%s -%s" .Cluster.ConfigItems.skipper_ingress_cpu .Cluster.ConfigItems.teapot_admission_controller_daemonset_reserved_cpu .Cluster.ConfigItems.kubelet_system_reserved_cpu) " " | sumQuantities }}
```

Reply:
Changed it based on your suggestions.